Packs/MalwareBazaar/Integrations/MalwareBazaar/README.md

```diff
@@ -765,3 +766,11 @@ Retrieves a list of recent malware samples (maximum 1000) associated with a spec
 >|---|---|---|---|---|---|---|---|
 >| ee566f0e04b497770c5baa4de14c416f | 06dac5f720847ff3c99c75a950a8b07dbf090127f770171f8d005a0c76c20de9 | 98f77caf24f14dca0fa163596a730269037f2065 | ee566f0e04b497770c5baa4de14c416f.exe | exe | 656384 | exe,<br/>TrickBot | 2021-11-07 08:38:51 |
 >| 7425c5e4bd0f910f80a1ab456b68e192 | 3d775f1f2da385ed73d988930d9c9675fc2e466098bb5f19c8501a723e14f437 | dbd26c846e03f94aa1451b6b73e1fa138ea5a953 | 7425c5e4bd0f910f80a1ab456b68e192.exe | exe | 656896 | exe,<br/>TrickBot | 2021-11-07 08:35:14 |
+
+
+#### Create an API Key
+1. Sign up for an abuse.ch account. You can do this easily by using an existing account that you may already have on X, LinkedIn, Google or Github. Just log in with the authentication provider of your choice here: https://auth.abuse.ch/
+2. Once you are authenticated on abuse.ch, ensure that you connect at least one additional authentication provider. This will ensure that you have access to our platforms, even if one of the authentication providers you use shuts down (yes, it happened with Twitter!)
```

Comment:
Once you've logged in to abuse.ch, add at least one more way to log in. This helps ensure you can always access our platforms, even if one of your login methods stops working.